pages/linux/flashrom.md

```diff
@@ -0,0 +1,24 @@
+# flashrom
+
+> Read, write, verify and erase flash chips.
+> More information: <https://man.archlinux.org/man/flashrom.8>.
+
+- Probe the chip, ensuring the wiring is correct:
+
+`flashrom -p {{programmer}}`
+
+- Read flash and save it into file:
+
+`flashrom -p {{programmer}} -r {{path/to/file}}`
+
+- Write file into flash:
+
+`flashrom -p {{programmer}} -w {{path/to/file}}`
+
+- Erase flash:
+
+`flashrom -p {{programmer}} -E`
+
+- Probe the chip using RaspberryPi, an example:
+
+`flashrom -p linux_spi:dev=/dev/spidev0.0`
```

Comment:
```suggestion
`flashrom --programmer {{linux_spi:dev=/dev/spidev0.0}}`
```

Reply:
@peterbabic I updated the above suggestion.

Reply:
I am still not sure what that handlebars do to be honest, but I agree.